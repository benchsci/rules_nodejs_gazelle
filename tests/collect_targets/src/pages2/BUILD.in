# gazelle:js_collect_all
# gazelle:js_collect_targets next_pages2
