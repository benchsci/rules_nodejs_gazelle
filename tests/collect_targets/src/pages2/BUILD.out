load("@aspect_rules_js//js:defs.bzl", "js_library")

# gazelle:js_collect_all
# gazelle:js_collect_targets next_pages2

js_library(
    name = "next_pages2",
    deps = ["//src/pages2/my_page"],
)
